persistence/account.go

```diff
@@ -51,9 +50,10 @@ func (p PostgresContext) SubtractAccountAmount(address []byte, amount string) er
 }
 
 // DISCUSS(team): If we are okay with `GetAccountAmount` return 0 as a default, this function can leverage
-//                `operationAccountAmount` with `*orig = *delta` and make everything much simpler.
+//
+//	`operationAccountAmount` with `*orig = *delta` and make everything much simpler.
```

Comment:
Please fix or revert this comment the spaces & new lines on this comment.

DItto in other places in the code as well.

Reply:
@Jasonyou1995 Make sure to click `Resolve conversation` if you've tended to the item and you do not expect any follow-up discussion (i.e. no other questions or controversies) related to it.